File: Python/Processor.py
class Processor:
    #atribut
    __name=""
    __price=0

    #constructor
    def __init__(self, name="", price=0):
        self.__name=name
        self.__price=price

    #Setter Getter
    def setName(self, name):
        self.__name=name

    def setPrice(self, price):
        self.__price=price

    def getName(self):
        return self.__name

    def getPrice(self):
        return self.__price

    #Method
    #Menampilkan atribut Processor
    def printProcessor(self):
        print("Processor Name: " + self.getName())
        print("Processor Price: " + str(self.getPrice()))

File: C++/Processor.cpp
#include <string>
#include <iostream>
using namespace std;

class Processor {
    private:
        //atribut
        string name;
        int price;
    public:
        //constructor
        Processor() {}
        Processor(string name, int price) {
            this->name=name;
            this->price=price;
        }
        //Setter Getter
        void setName(string name) { this->name=name; }
        void setPrice(int price) { this->price=price; }

        string getName() { return this->name; }
        int getPrice() { return this->price; }

        //Method
        void printProcessor() {
            cout<<"Processor Name: "<<this->name<<endl;
            cout<<"Processor Price: "<<this->price<<endl;
        }
        //Destructor
        ~Processor() {}
};